src/System.Xml.XDocument/src/System/Xml/Linq/OrderedNodeEnumerable.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
I did start with the file that got deleted in this (git doesn't show that well unless there's a commit that only moves a file), so its definitely a derived work that started with something copyright Microsoft, and hence they are a copyright-holder on the derived work. The changes are pretty substantial so even when not listing all contributors (as is not done on this project) one could make an argument for *adding* me as a listed copyright holder on the file, but not for replacing MS entirely, as far as I can see.